diff --git a/tests/data/keep-2-files-in-sync/a.sh b/tests/data/keep-2-files-in-sync/a.sh
index f4ba23f..435dfe7 100644
--- a/tests/data/keep-2-files-in-sync/a.sh
+++ b/tests/data/keep-2-files-in-sync/a.sh
@@ -1,5 +1,6 @@
 export LOG="thumbnail=debug;video=info;"
 # if-change
+export THUMBNAIL_BUCKET="s3://video-thumbnails/"
 export VIDEO_BUCKET="s3://video-service/"
 # then-change tests/data/keep-2-files-in-sync/b.sh
 export VIDEO_CONFIG="video-service/config.json"
