diff --git a/tests/data/basic/z.sh b/tests/data/basic/z.sh
index 771ac22..dc8ee1c 100644
--- a/tests/data/basic/z.sh
+++ b/tests/data/basic/z.sh
@@ -2,5 +2,6 @@ echo "starting watchdog thread"
 echo "starting handler for /health"
 # if-change
 echo "starting handler for /image-api/v1"
+echo "starting handler for /thumbnail-api/v1"
 echo "starting handler for /video-api/v1"
 # then-change nonexistent.cfg
